ui/src/features/Editor/components/BottomPanel/components/Map/index.tsx

```
@@ -2,17 +1,24 @@
-const sceneModes: SceneMode[] = ["2d", "3d"];
+export type MapMode = "2d" | "3d";
 
-const Map: React.FC = () => {
+const mapModes: MapMode[] = ["2d", "3d"];
+
+type Props = {
+  mapMode: MapMode;
+  setMapMode?: (mode: MapMode) => void;
+};
+
+const Map: React.FC<Props> = ({ mapMode, setMapMode }) => {
```

Biome

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)